App/Models/Http/Responses/SubStatusRes.cs

```diff
@@ -0,0 +1,12 @@
+﻿
+using Newtonsoft.Json;
+
+namespace GamHubApp.Models.Http.Responses;
+
+public sealed class SubStatusRes
+{
+    [JsonProperty("message")]
+    public string msg { get; set; }
+    [JsonProperty("Enabled")]
```

Comment:
Need to change this to lowercase, we left this to compensate as the API currently returns is in uppercase.
Either way it's getting fixed now so it won't matter for long


```suggestion
    [JsonProperty("enabled")]
```